packages/xo-server/src/api/host.mjs

```diff
@@ -530,6 +531,30 @@ setControlDomainMemory.resolve = {
   host: ['id', 'host', 'administrate'],
 }
 
+// -------------------------------------------------------------------
+
+export async function isPubKeyTooShort({ host }) {
+  const certificate = await this.getXapi(host).callAsync('host.get_server_certificate', host._xapiRef)
```

Comment:
IMHO, this is long running action but a simple get, you can use `call` instead of `callAsync` :slightly_smiling_face: